libvast/CMakeLists.txt

```diff
@@ -236,6 +242,56 @@ install(
 
 add_library(vast::libvast ALIAS libvast)
 
+# ----------------------------------------------------------------------------
+# flattbuffers
+# ----------------------------------------------------------------------------
+
+file(GLOB flatbuffers_schemas ${CMAKE_CURRENT_SOURCE_DIR}/vast/fbs/*.fbs)
```

Comment:
Globbing means you won't touch the CMakeLists.txt when a new file was added. This might be convenient at first sight, but it also means that existing build directories can't know when a regeneration is needed. This will result in failing builds for anyone that updates master after such an addition was merged.

I recommend to list these files explicitly instead.

Reply:
I think adding `CONFIGURE_DEPENDS` on the GLOB would be fine as well.